src/vmm/src/linux/vstate.rs

```diff
@@ -927,7 +1035,12 @@ impl Vcpu {
     /// * `exit_evt` - An `EventFd` that will be written into when this vcpu exits.
     /// * `create_ts` - A timestamp used by the vcpu to calculate its lifetime.
     #[cfg(target_arch = "aarch64")]
-    pub fn new_aarch64(id: u8, vm_fd: &VmFd, exit_evt: EventFd) -> Result<Self> {
+    pub fn new_aarch64(
+        id: u8,
+        vm_fd: &VmFd,
+        exit_evt: EventFd,
+        sender_io: Sender<MemProperties>,
```

Comment:
should be CCA only

Reply:
The memory fault exception isn't TEE-specific (although it could be used that way), so I don't think I agree that this should be CCA only.

Reply:
You're right. I did this part of the review before you and I had talked about the order of patches.

Reply:
I think we can enable the extra this extra thread only when the `tee` feature is enabled.